hack/generate-kcp-api.sh

```diff
@@ -52,6 +52,25 @@ kind: APIExport
 metadata:
   name: release
 spec:
+  permissionClaims:
+  - resource: "applications"
+    group: "appstudio.redhat.com"
+    #identityHash: "Get value from apiexport. It is not static" # has apiexport
```

Comment:
Are this comments needed? I don't really get them btw. Not sure what they mean, but that's probably caused by my lack of knowledge on KCP.

In any case, I wouldn't add them like this. I would have an external file defined in... probably `config/kcp` with a list of the `permissionClaims` we need and then in the script process that file. It's cleaner and that way people could copy the script if they want.

Reply:
Well, if you don't define the `identityHash`, it won't work. However, the values are essentially uuids. Nothing static I can put here will work. I opened a slack thread about this yesterday, I'll send it to you